### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Victorayo**
I added both IPI power Tech and KITS Technologies as part of the submission for the tech companies. Kindly merge

### CODE REVIEW

**Comment by OSCA-Ado-Ekiti** [comment]
@Victorayo Kindly resolve the conflicts in the README.md file

**Comment by Victorayo** [comment]
> @Victorayo Kindly resolve the conflicts in the README.md file

I've resolved the issue. 

I think it was created by someone else trying to add Interswitch to the list. But it's been corrected now.

You can merge my request now pls

File: README.md [hunk 1]
```diff
@@ -342,10 +342,20 @@
 
   - Founder: [@mrmakoko](https://twitter.com/mrmakoko)
 
-  - Industry: Venture Capital
+
```
Comment (gray-adeyi) [inline README.md:345]: Please also include this information in the `techCompanies.js`

**Review by gray-adeyi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Victorayo) [reply README.md:345]: Done that already in #125 

@gray-adeyi @OSCA-Ado-Ekiti

Comment (gray-adeyi) [reply README.md:345]: Okay thanks. I'll review it now

**Pull request merged**